ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -275,6 +382,60 @@ export interface Data extends RaidbossData {
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM4Savage',
   zoneId: ZoneId.AacLightHeavyweightM4Savage,
+  config: [
+    {
+      id: 'ionCluster',
+      name: {
+        en: 'Ion Cluster Debuff Strategy',
+      },
+      comment: {
+        en: 'Strategy for resolving debuffs during Ion Cluster.',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'None - Just call your debuff': 'none',
+          'DN - Call rivet positions (Support left, DPS right)': 'DN',
+        },
+      },
+      default: 'none',
+    },
+    {
+      id: 'witchHunt',
+      name: {
+        en: 'Witch Hunt Bait Strategy',
+      },
+      comment: {
+        en: 'Strategy for baiting Witch Hunt AoEs',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'None - Just call party and bait player positions.': 'none',
+          'DN - Supports take first and second baits, prioritizing uptime.': 'DN',
+        },
+      },
+      default: 'none',
+    },
+    {
+      id: 'sunrise',
+      name: {
+        en: 'Sunrise Sabaath Strategy',
+      },
+      comment: {
+        en: 'Strategy for resolving Sunrise Sabaath',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'None - Just call debuff and both tower spawns.': 'none',
+          'Uptime - Call specific cannon and tower spots using priority system (DPS N and CW, Support NW and CCW). Assumes AutoCAD uptime waymarks.':
```

Comment:
Not going to be able to look at this for a while, but on a quick look, this caught my eye -- the prop key is what goes in the select box, and this is a long string to have inside a dropdown.  For this and the others, I'd suggest much shorter names, and then use the additional property (I think it's `comment`?) to add an explanatory comment and link to the strats -- see p12s for examples.

Reply:
Sure. I wasn't quite sure how to explain, since I wanted to include enough information for folks to find and ensure they are using the right strategy. These are somewhat specific assumptions about order and spots.

Reply:
Oh, yeah I think explanations are super helpful - definitely not suggesting to remove them or anything.  I think `comment` was added for just that reason so we could explain/link to strat options & explanations.